var/spack/repos/builtin/packages/ramses/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Ramses(MakefilePackage):
+    """Ramses benchmark for DiRAC.
+
+    The source code for this benchmark is stored in a private repository. To
+    gain access please contact the RSE team at the University of Leicester or
+    contact via github from our organization page
+    https://github.com/UniOfLeicester
+    """
+
+    homepage = "https://github.com/UniOfLeicester/benchmark-ramses"
+    git = "ssh://git@github.com/UniOfLeicester/benchmark-ramses.git"
+
+    maintainers = ["TomMelt"]
+
+    version("v1.0.0", branch="main")
```

Comment:
Numbered versions are associated with either a release, tag, or commit.  Perhaps you just want to add a branch version?

```suggestion
    version("main", branch="main")
```

See https://spack.readthedocs.io/en/latest/packaging_guide.html#git.

Reply:
thanks, I have changed the version to use git tags instead